src/droplet.F

```diff
@@ -302,9 +319,27 @@ subroutine droplet_driver(dt,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,    &
     !$acc loop gang vector
 #endif
 
+    t_f = mpi_wtime()
+    if (myid==0) write(*,*) 'TIME prep:',t_f-t_s
+
+    t_s = mpi_wtime()
+
+    t_findloc = 0.0
+    t_interp = 0.0
+    t_BEint = 0.0
+    t_feed = 0.0
+
+    t_BEprep = 0.0
+    t_radsolver = 0.0
+    t_gauss = 0.0
+    t_LV = 0.0
+    t_neighbor = 0.0
+
```

Comment:
These changes will break the GPU code. Need to be moved up to line 311.

Reply:
These were all for my timing tests, I have removed them